.circleci/config.yml

```diff
@@ -65,11 +65,11 @@ jobs:
             rm -rf build/librem_l1um/* build/log/* && make CPUS=4 V=1 BOARD=librem_l1um || touch /tmp/failed_build
           no_output_timeout: 3h
       - run:
-          name: Output build failing logs 
+          name: Ouput build failing logs 
```

Comment:
@tlaurion  Search "Ouput" replace with "Output" in `.circleci.yaml`.  There are multiple instances.